### ISSUE TITLE
terraform import for aws_backup_plan?

### ISSUE THREAD

**Issue by ricardclau** [terraform-providers/terraform-provider-aws#12380]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

I am in the process of terraform importing a relatively big setup that was done manually in the Console and so far so good but I have found that aws_backup_plans do not have import functionality supported

There is probably a good reason for this but I could not find it after navigating through issues and general googling so any light here would be much appreciated

Other aws_backup_* resources support importing this is why I am asking :)

### New or Affected Resource(s)

* aws_backup_plan

### Potential Terraform Configuration

```hcl
resource "aws_backup_plan" "foo" {
    name = "my_backup_plan

   /* List of rules here */
}
```

```bash
terraform import aws_backup_plan.foo my_backup_plan
```

### References

Could not find any issues related to this so apologies if there is one open that I missed

**Comment by bflad** [comment]
Support for `aws_backup_plan` resource import has been merged and will release with version 2.54.0 of the Terraform AWS Provider, later this week. Thanks to @DrFaust92 for the implementation. 👍

**Comment by ricardclau** [comment]
Awesome! Thank you so much!
